### DISCUSSION TITLE
OMEdit error message

### DISCUSSION THREAD

**Discussion by ZackeyZAKI** [OpenModelica/OpenModelica#10751] [Q&A]
I apologize, but I still need the specific error message you encountered in order to provide an accurate translation and appropriate solution. Could you please provide the error message that appeared in OMEdit?

[Buildings.Fluid.Sources.BaseClasses.PartialSource: 50:5-55:3]: Assertion triggered during translation: "
Each ports[i] of boundary shall at most be connected to one component.
If two or more connections are present, ideal mixing takes
place in these connections, which is usually not the intention
of the modeller. Increase nPorts to add an additional port.
".

**Comment by casella** [comment]
@ZackeyZAKI this obviously refers to some previous discussion item. Can you please link it here. Otherwise we lack a bit of context 😅
